static/app/utils/onDemandMetrics/index.tsx

```diff
@@ -121,14 +129,43 @@ export function hasOnDemandMetricWidgetFeature(organization: Organization) {
   );
 }
 
-export function OnDemandWarningIcon({msg}: {msg: React.ReactNode}) {
+export function OnDemandWarningIcon({
+  msg,
+  color = 'gray300',
+}: {
+  msg: React.ReactNode;
+  color?: Color;
+}) {
   return (
     <Tooltip title={msg}>
-      <HoverableIconWarning color="gray300" />
+      <HoverableIconWarning color={color} />
     </Tooltip>
   );
 }
 
+const InfoAlert = styled(Alert)`
+  border: 1px solid ${p => p.theme.blue400};
+
+  & > span {
+    display: flex;
+    align-items: center;
+  }
+`;
+
+const InfoICon = styled(IconInfo)`
+  color: ${p => p.theme.blue400};
+  margin-right: ${space(1.5)};
+`;
+
+export function OnDemandMetricAlert({message}: {message: React.ReactNode}) {
+  return (
+    <InfoAlert>
+      <InfoICon size="sm" />
+      {message}
+    </InfoAlert>
+  );
+}
```

Comment:
nit: from what I have seen we tend to keep the styled wrapper components at the bottom and every other component above them.
Therefore, I would move this one right under ` OnDemandWarningIcon`.

Reply:
👍